### DISCUSSION TITLE
Is the heading40 fontSize variance intentional between Evergreen and the other Paste themes?

### DISCUSSION THREAD

**Discussion by ed-twilion** [twilio-labs/paste#3734] [General] [answered]
I noticed that Twilio Paste's Evergreen theme has the font size for `heading40` set at `0.875rem` (same as `heading50`), however, for every other theme `heading40` fontSize is `1rem`. Is this intentional?

![image](https://github.com/twilio-labs/paste/assets/99840997/16394290-638d-405d-9a9d-ee67b4a97b33)

**Comment by serifluous** [comment]
Hey @ed-twilion ! We're checking in with the designers on Segment right now to check whether this was intentional or not.

**Reply by serifluous** [reply, marked as answer]
We've got a fix up for this here: #3735. Thanks for catching this!

**Reply by SiTaggart** [reply]
A fix was released in Core v20.8.0
